polaris.shopify.com/src/components/Markdown/Markdown.tsx

```diff
@@ -30,10 +30,10 @@ import {FeaturedCardGrid} from '../FeaturedCardGrid';
 import {useCopyToClipboard} from '../../utils/hooks';
 import {Colors} from './components/Colors';
 
-const CodeVisibilityContext = createContext<[boolean, (arg: boolean) => void]>([
-  false,
-  () => {},
-]);
+const CodeVisibilityContext = createContext<{
+  showCode?: boolean;
```

Comment:
We allow these values to be conditionally undefined, as otherwise the checks below would **always** inherit the default context values, and our setter `setShowCode` would always be a noop